tests/main/res/TestIIDM_hvdc_dangling.iidm

```diff
@@ -0,0 +1,72 @@
+      <lccConverterStation id="LCCStation1" name="S1" lossFactor="0" powerFactor="0" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="-100" q="-0"/>
+      <vscConverterStation id="VSCStation1" name="VSCS1" voltageRegulatorOn="true" lossFactor="0" voltageSetpoint="400" reactivePowerSetpoint="0" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="-100" q="0">
+        <reactiveCapabilityCurve>
+          <point p="-1011" minQ="-300" maxQ="300"/>
+          <point p="0" minQ="-600" maxQ="400"/>
+          <point p="1011" minQ="-300" maxQ="300"/>
+        </reactiveCapabilityCurve>
+      </vscConverterStation>
+    </voltageLevel>
+  </substation>
+  <substation id="_BUS___11_SS" name="BUS   11_SS" country="AF">
+    <voltageLevel id="_BUS___11_VL" name="BUS   11_VL" nominalV="400" topologyKind="BUS_BREAKER">
+      <busBreakerTopology>
+        <bus id="_BUS___11_TN" v="400" angle="0"/>
+      </busBreakerTopology>
+      <lccConverterStation id="LCCStation2" name="S2" lossFactor="0" powerFactor="0" bus="_BUS___11_TN" connectableBus="_BUS___11_TN" p="100" q="-0"/>
+      <vscConverterStation id="VSCStation2" name="VSCS2" voltageRegulatorOn="false" lossFactor="0" voltageSetpoint="400" reactivePowerSetpoint="0" bus="_BUS___11_TN" connectableBus="_BUS___11_TN" p="100" q="0">
+        <reactiveCapabilityCurve>
+          <point p="-1011" minQ="-300" maxQ="300"/>
+          <point p="0" minQ="-600" maxQ="400"/>
+          <point p="1011" minQ="-300" maxQ="300"/>
+        </reactiveCapabilityCurve>
+      </vscConverterStation>
+    </voltageLevel>
+  </substation>
+  <line id="_BUS____1-BUS____5-1_AC" name="BUS    1-BUS    5-1" r="0.01" x="0.050000000000000003" g1="0" b1="0.00051670000000000004" g2="0" b2="0.00051670000000000004" voltageLevelId1="_BUS____1_VL" bus1="_BUS____1_TN" connectableBus1="_BUS____1_TN" voltageLevelId2="_BUS____5_VL" bus2="_BUS____5_TN" connectableBus2="_BUS____5_TN" p1="900.0898860722657" q1="-165.03365280803723" p2="-900.03882699445762" q2="-0.050012760451512671">
+    <currentLimits1 permanentLimit="836.74000000000001"/>
+  </line>
+  <line id="_BUS____5-BUS___10-1_AC" name="BUS    5-BUS   10-1" r="0.01" x="0.050000000000000003" g1="0" b1="0" g2="0" b2="0" voltageLevelId1="_BUS____5_VL" bus1="_BUS____5_TN" connectableBus1="_BUS____5_TN" voltageLevelId2="_BUS___10_VL" bus2="_BUS___10_TN" connectableBus2="_BUS___10_TN" p1="400.03884322185792" q1="0.050012760310907206" p2="-400.02884066976839" q2="1.3706019826031923e-10">
+    <currentLimits1 permanentLimit="4183.6999999999998"/>
+  </line>
+  <hvdcLine id="HVDCLCCLine" r="0.02" nominalV="400" convertersMode="SIDE_1_INVERTER_SIDE_2_RECTIFIER" activePowerSetpoint="100" maxP="2000" converterStation1="LCCStation1" converterStation2="LCCStation2"/>
+  <hvdcLine id="HVDCVSCLine" r="0.02" nominalV="400" convertersMode="SIDE_1_INVERTER_SIDE_2_RECTIFIER" activePowerSetpoint="100" maxP="2000" converterStation1="VSCStation1" converterStation2="VSCStation2"/>
```

Comment:
Can you inverse one line ?
You can write converterStation1="VSCStation2" converterStation2="VSCStation1" instead of converterStation1="VSCStation1" converterStation2="VSCStation2"